### ISSUE TITLE
[18.10] Firefox and full screen button

### ISSUE THREAD

**Issue by garnier-quentin** [centreon/centreon#7222]
<h1>BUG REPORT INFORMATION </h1>

<h3> Prerequisites </h3>

***Versions***

For the RPM based systems

-- Copy/Paste the result of the following command --
```
$ rpm -qa | grep centreon
centreon-plugin-Network-Firewalls-Juniper-Ssg-Snmp-20190111-144402.el7.centos.noarch
centreon-plugin-Applications-Protocol-Ftp-20190111-102147.el7.centos.noarch
centreon-plugin-Network-Cisco-Standard-Snmp-20190111-102147.el7.centos.noarch
centreon-widget-servicegroup-monitoring-18.10.0-6.el7.centos.noarch
centreon-plugins-base-1.18-2.el7.centos.noarch
centreon-license-manager-common-18.10.1-2.el7.centos.noarch
centreon-widget-live-top10-cpu-usage-18.10.0-6.el7.centos.noarch
centreon-plugin-Hardware-Ups-Standard-Rfc1628-Snmp-20190111-102147.el7.centos.noarch
centreon-broker-18.10.1-1.el7.centos.x86_64
centreon-engine-extcommands-18.10.0-4.el7.centos.x86_64
centreon-connector-18.10.0-2.el7.centos.x86_64
centreon-web-18.10.2-3.el7.centos.noarch
centreon-18.10.2-3.el7.centos.noarch
centreon-awie-18.10.0-3.el7.centos.noarch
centreon-plugin-Applications-Monitoring-Centreon-Database-20190111-102147.el7.centos.noarch
centreon-widget-live-top10-memory-usage-18.10.0-6.el7.centos.noarch
centreon-plugin-Applications-Protocol-Http-20190111-102147.el7.centos.noarch
centreon-plugin-Applications-Monitoring-Centreon-Map4-Jmx-20190111-102147.el7.centos.noarch
centreon-widget-tactical-overview-18.10.0-6.el7.centos.noarch
centreon-plugin-Operatingsystems-Windows-Snmp-20190111-102147.el7.centos.noarch
centreon-plugin-Virtualization-VMWare-daemon-2.4.0-1.el7.centos.noarch
centreon-broker-storage-18.10.1-1.el7.centos.x86_64
centreon-engine-daemon-18.10.0-4.el7.centos.x86_64
centreon-engine-18.10.0-4.el7.centos.x86_64
centreon-broker-cbd-18.10.1-1.el7.centos.x86_64
centreon-poller-centreon-engine-18.10.2-3.el7.centos.noarch
centreon-base-config-centreon-engine-18.10.2-3.el7.centos.noarch
centreon-pp-manager-18.10.2-1.el7.centos.noarch
centreon-plugin-Operatingsystems-Linux-Snmp-20190111-102147.el7.centos.noarch
centreon-plugin-Hardware-Printers-Generic-Snmp-20190111-102147.el7.centos.noarch
centreon-plugin-Applications-Protocol-Dns-20190111-102147.el7.centos.noarch
centreon-widget-host-monitoring-18.10.0-11.el7.centos.noarch
centreon-plugin-Applications-Monitoring-Centreon-Central-20190111-102147.el7.centos.noarch
centreon-plugin-Applications-Monitoring-Centreon-Poller-20190111-102147.el7.centos.noarch
centreon-widget-engine-status-18.10.0-9.el7.centos.noarch
centreon-common-18.10.2-3.el7.centos.noarch
centreon-broker-core-18.10.1-1.el7.centos.x86_64
centreon-perl-libs-18.10.2-3.el7.centos.noarch
centreon-broker-cbmod-18.10.1-1.el7.centos.x86_64
centreon-connector-ssh-18.10.0-2.el7.centos.x86_64
centreon-license-manager-18.10.1-2.el7.centos.noarch
centreon-widget-grid-map-18.10.0-6.el7.centos.noarch
centreon-plugin-Applications-Protocol-Ldap-20190111-102147.el7.centos.noarch
centreon-plugin-Applications-Databases-Mysql-20190111-102147.el7.centos.noarch
centreon-clib-18.10.0-2.el7.centos.x86_64
centreon-trap-18.10.2-3.el7.centos.noarch
centreon-connector-perl-18.10.0-2.el7.centos.x86_64
centreon-widget-graph-monitoring-18.10.0-6.el7.centos.noarch
centreon-widget-service-monitoring-18.10.0-11.el7.centos.noarch
centreon-widget-hostgroup-monitoring-18.10.0-6.el7.centos.noarch
centreon-plugin-Cloud-Aws-Ec2-Api-20190111-102147.el7.centos.noarch
centreon-release-18.10-2.el7.centos.noarch
centreon-database-18.10.2-3.el7.centos.noarch
```

***Operating System***
Centos 7

***Additional environment details (AWS, VirtualBox, physical, etc.):***

<h3> Description </h3>

When i click on the screen button, nothing happen. Moreover, the menu is not working (when i click on a menu, i stay on the same page...)

<h3> Steps to Reproduce </h3>

Please describe precisely the steps to reproduce the encountered issue.

1. I logged in Centreon
2. I click on fullscreen button

<h3> Describe the received result </h3>

When i click on the screen button, nothing happen. Moreover, the menu is not working (when i click on a menu, i stay on the same page...)

<h3> Describe the expected result </h3>

it's working.

<h3> Additional relevant information (e.g. frequency, ...) </h3>

Firefox 65.0

**Comment by garnier-quentin** [comment]
I can also give webconsole errors:
```
warning: Request for fullscreen was denied because Element.requestFullscreen() was not called from inside a short running user-generated event handler in  index.js:49
critical: TypeError: The expression cannot be converted to return the specified type. 1.c455e50c.chunk.js:1
```

**Comment by cgagnaire** [comment]
Hi,
Still present in 19.04.3 with same message (French flavor) :

> La requête d’accès au plein écran a été refusée, car Element.requestFullScreen() n’a pas été appelée à l’intérieur d’un gestionnaire d’évènement généré par l’utilisateur. vendors~main.fa07f8d1.chunk.js:1:988226,

**Comment by BenoitPoulet** [comment]
As workaround, for fullscreen mode you should hide :
- left bar
- bottom bar
- top bar

The real full screen can be managed in the browser via extension or by the user.

**Comment by cgagnaire** [comment]
Fixed in 19.04.x and 19.10.x by updating a buggy react dep
